bazel/repository_locations.bzl

```diff
@@ -322,26 +322,26 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_desc = "Data Collect Protocols of Apache SkyWalking",
         project_url = "https://github.com/apache/skywalking-data-collect-protocol",
         name = "skywalking_data_collect_protocol",
-        sha256 = "fa9ac679624217f30b6e8d5c450365386c610e2d08188a20f0340c3b14401833",
-        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/v8.3.0.zip"],
-        strip_prefix = "skywalking-data-collect-protocol-8.3.0",
-        version = "8.3.0",
+        sha256 = "4a3401098e0e49ae56d03cc4890b21c75bab6ccf1c157d73bb0c93fc15bb07c7",
+        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/{version}.tar.gz"],
+        strip_prefix = "skywalking-data-collect-protocol-{version}",
+        version = "c987be061e570ce7d0d8f415753f05555e787407",
         use_category = ["observability_ext"],
         extensions = ["envoy.tracers.skywalking"],
-        release_date = "2020-11-20",
+        release_date = "2021-02-02",
         cpe = "N/A",
     ),
     com_github_skyapm_cpp2sky = dict(
         project_name = "cpp2sky",
         project_desc = "C++ SDK for Apache SkyWalking",
         project_url = "https://github.com/SkyAPM/cpp2sky",
-        sha256 = "a8d870bb4b1c4a05eae319f689d1948927f3f0a5b5fe524db73a4c04121a339a",
-        version = "0.1.1",
+        sha256 = "7122b8e2ffee33d16998e767d7971906aa88826c16685842ab8a93378cd910b0",
+        version = "a4971da76507c0cb8199071c212867e1d2558c0c",
```

Comment:
0.2.0 not release yet? https://github.com/SkyAPM/cpp2sky/releases

Reply:
Once I released v0.2.0 but several problems had occurred while Envoy integration. So we will release it again after pending PR had merged, then ready to merge this.